update services articles in v14 (#5987)
umbraco/UmbracoDocs: 14/umbraco-cms/reference/management/services/create-a-new-user.md

---
description: This will show you how to add a user to a user group using the UserService in Umbraco.
---

# Getting the service

## Services property

> If you wish to use the UserService in a class that inherits from one of the Umbraco base classes (eg. `SurfaceController`, `UmbracoApiController` or `UmbracoAuthorizedApiController`), you can access the service through a local `Services` property:

Reviewer: ```suggestion
If you wish to use the UserService in a class that inherits from one of the Umbraco base classes (For example: `SurfaceController`, `UmbracoApiController`, or `UmbracoAuthorizedApiController`), you can access the service through a local `Services` property:
```

```csharp
IUserService userService = Services.UserService;
```

## Static accessor

> If neither a `Services` property or Dependency Injection is available, you can also reference the static `Current` class directly:

Reviewer: ```suggestion
If neither a `Services` property nor Dependency Injection is available, you can reference the static `Current` class directly:
```
